data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1847 @@ mission "Stone of our Fathers 5"
+					goto ice3
+			label researcher3
+			`	"Long time ago. Researcher convoy look for bacterial pharmaceutical. Many researcher convoy in area. Most smart use native pathfinder, like me. This convoy, they not so smart, they say pathfinder waste of money. They go out in ice alone and they never come back."`
+			`	Hjlod shrugs. "Maybe they fall in ice. Maybe they get eaten by Ice-Slug. Maybe avalanche fall on them. Maybe fall on own veapon and die. Nobody ever find them. Tour-guide book say that some night you can see their ghost haunting ice. Tour-guide book is dumb."`
+			`	"You have more question?"`
+			choice
+				`	"Nope. No questions."`
+					goto noquestion
+				`	"Is it safe to drive over the ice?"`
+					goto ice31
+			label ice31
+			`	"Very safe. Except when not. Ice always frozen, but sometime very thin. I can tell. You cannot. Trust Hjlod and you not drown and get eaten by Ice-Slug."`
+			`	"Now, no more question." With that Hjlod swivels back into her cockpit and begins plotting a new course for the Ice-Crawler.`
+			scene "scene/iceplains2"
+			choice
+				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
+					goto rock
+			label ice3
+			`	"Very safe. Except when not. Ice always frozen, but sometime very thin. I can tell. You cannot. Trust Hjlod and you not drown and get eaten by Ice-Slug."`
+			`	"You have more question?"`
+			choice
+				`	"Nope. No questions."`
+					goto noquestion
+				`	"What happened to the researchers?"`
+					goto researcher31
+			label researcher31
+			`	"Long time ago. Researcher convoy look for bacterial pharmaceutical. Many researcher convoy in area. Most smart use native pathfinder, like me. This convoy, they not so smart, they say pathfinder waste of money. They go out in ice alone and they never come back."`
+			`	Hjlod shrugs. "Maybe they fall in ice. Maybe they get eaten by Ice-Slug. Maybe avalanche fall on them. Maybe fall on own veapon and die. Nobody ever find them. Tour-guide book say that some night you can see their ghost haunting ice. Tour-guide book is dumb."`
+			`	"Now, no more question." With that Hjlod swivels back into her cockpit and begins plotting a new course for the Ice-Crawler.`
+			scene "scene/iceplains2"
+			choice
+				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
+					goto rock
+			label noquestion
+			`	"Is smart, not to ask question. Hjlod tell you best parts only." Hjlod gives you a quick grin and swivels back into her makeshift cockpit.`
+			`	"Enough ice-plain, now ve go to next stop."`
+			scene "scene/iceplains2"
+			choice
+				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
+					goto rock
+			label rock
+			`	The Ice-Crawler plows through snow, hill, mountain, and tundra for another hour or two before a massive and oddly shaped rock passes into view. Hjlod stops the Ice-Crawler so you can look at it.`
```

Comment:
```suggestion
			`	Hjlod slows down and looks back at you. "You have question?"`
			label questions
			choice
				`	"Nope. No questions."`
					goto noquestion
				`	"What happened to the researchers?"`
					goto researcher
				`	"Is it safe to drive over the ice?"`
					goto ice
				`	"Are there sea-monsters beneath the ice?"`
					goto monster
			label researcher
			`	"Long time ago. Researcher convoy look for bacterial pharmaceutical. Many researcher convoy in area. Most smart use native pathfinder, like me. This convoy, they not so smart, they say pathfinder waste of money. They go out in ice alone and they never come back."`
			`	Hjlod shrugs. "Maybe they fall in ice. Maybe they get eaten by Ice-Slug. Maybe avalanche fall on them. Maybe fall on own veapon and die. Nobody ever find them. Tour-guide book say that some night you can see their ghost haunting ice. Tour-guide book dumb."`
			`	"Any more question?"`
				goto questions
			label ice
			`	"Very safe. Except when not. Ice always frozen, but sometime very thin. I can tell. You cannot. Trust Hjlod and you not drown and get eaten by Ice-Slug."`
			` 	"You have more question?"`
				goto questions
			label monster
			`	"Every world have animal. Some animal big, some small. Voolly Octopus, mostly small. Size of this Ice-Crawler, only. Vorst in Glaesisvellir ice-plains is Ice-Slug. They can squeeze through tiniest hole in ice. This take long time, for they are as big as house. They have no teeth, too hard to squeeze through little hole, but they have acid spit. Put hole in Ice-Crawler, easy. They eat many foolish researcher and tourist."`
			`	She smiles at you. "You're not foolish tourist. You are with Hjlod. You are safe."`
			`	"Any more question?"`
				goto questions
			label noquestion
			`	"Is smart, not to ask question. Hjlod tell you best parts only." Hjlod gives you a quick grin and swivels back into her makeshift cockpit.`
			`	"Enough ice-plain, now ve go to next stop."`
			scene "scene/iceplains2"
			choice
				`	(You look out at the bright Ice-Plains as they slowly disappear into the dull horizon.)`
			`	The Ice-Crawler plows through snow, hill, mountain, and tundra for another hour or two before a massive and oddly shaped rock passes into view. Hjlod stops the Ice-Crawler so you can look at it.`
```
This provides a label right before the questions, which you loop back to after asking any question. This means you can't exhaust every option like you could before, but it results in a much cleaner script.